### PR TITLE
Added Source definition in Process Group

### PR DESCRIPTION
**Pull request by MichalVacek**
# Summary
* Added Source definition in Process Group
* Deprecated Channel in Process Group

### CODE REVIEW

**Review by patrikmiksovsky95 — APPROVED** [review]

File: changelog/changelog2022.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Changelog 2022
+
+## 19th September 2022
+
+* [Source](../mews-operations/reservations.md#source) replaced Channel in the [Process Group Request](../mews-operations/reservations.md#request), which now supports processing of multiple sources.
+* [Travel Agency](../mews-operations/reservations.md#company) was added to [Process Group Request](../mews-operations/reservations.md#request).
```
Comment (patrikmiksovsky95) [inline changelog/changelog2022.md:6]: wrong link to travel agency

File: mews-operations/reservations.md [hunk 2]
```diff
@@ -82,6 +91,21 @@ The third `reservation` definition shows the partial cancellation - cancelling t
...
+        "name": "Best Travel Agency, s.r.i",
+        "contact": {
+            "email": "best@company.com",
+            "phone": "+420 775 775 775",
+            "address": {
+                "zip": "132 45",
+                "addressLine1": "Some other street 123",
+                "addressLine2": "Some other detail",
+                "city": "Some other city",
+                "country": "US",
```
Comment (patrikmiksovsky95) [inline mews-operations/reservations.md:105]:
```suggestion
                "country": "US"
```

File: mews-operations/reservations.md [hunk 3]
```diff
@@ -82,6 +91,21 @@ The third `reservation` definition shows the partial cancellation - cancelling t
             }
         }
     },
+    "travelAgency": {
+        "iata": "65553",
+        "name": "Best Travel Agency, s.r.i",
```
Comment (patrikmiksovsky95) [inline mews-operations/reservations.md:96]:
```suggestion
        "name": "Best Travel Agency, s.r.o",
```

File: changelog/changelog2022.md (hunk 1 above)
Comment (MikeAdamsMews) [reply changelog/changelog2022.md:6]: Is it true that the meaning of Company has changed? I'm not sure myself, but I suspect in the old API, 'Company' could mean either a Travel Agent or a Corporate, now this has been split into two, so 'Company' now only ever means a Corporate, and a new property has been added 'Travel Agency' which means a Travel Agent. I hope I'm making sense!

**Review by MikeAdamsMews — CHANGES_REQUESTED** [review]
There's a few small details that need correcting before it can be approved.

**Review by patrikmiksovsky95 — APPROVED** [review]

File: mews-operations/reservations.md [hunk 4]
```diff
@@ -288,12 +322,29 @@ The third `reservation` definition shows the partial cancellation - cancelling t
...
 | `13` | Maestro |
 
-#### Channel
+#### Source
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `code` | `int` | required | [Channel](#channels) code. |
-| `name` | `string` | required | Name of the Channel. |
+| `code` | `int` | required \(exc. new unknown sources\) | [Source](#source) code. |
```
[the remark this answers is not in the document]
Comment (MikeAdamsMews) [reply mews-operations/reservations.md:329]: I think the Contract should be "required \(with the exception of new, unknown sources\)". I think the Description should be "Channel code, see [Channels](../channels/README.md)". I know the plan is to remove the list of channels from the docs, but at the moment that's where they are listed.

File: mews-operations/reservations.md [hunk 5]
```diff
@@ -288,12 +322,29 @@ The third `reservation` definition shows the partial cancellation - cancelling t
...
+| `name` | `string` | required | Name of the [Source](#source). |
+| `type` | `int` | required | [Source Type](#source-types) code of the [Source](#source). |
```
Comment (MikeAdamsMews) [inline mews-operations/reservations.md:331]: Just "[Source Type](#source-types) code" is sufficient.

File: mews-operations/reservations.md [hunk 6]
```diff
@@ -288,12 +322,29 @@ The third `reservation` definition shows the partial cancellation - cancelling t
...
+| `isPrimary` | `bool` | required | Mark the primary source where the reservation group comes from. |
```
Comment (MikeAdamsMews) [inline mews-operations/reservations.md:332]: I would say "Indicates which source is the primary source for the reservation group"

File: changelog/changelog2022.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+# Changelog 2022
+
+## 19th September 2022
+
+* [Sources](../mews-operations/reservations.md#source) replaces Channel in the [Process Group Request](../mews-operations/reservations.md#request), which now supports processing of multiple sources.
```
Comment (MikeAdamsMews) [inline changelog/changelog2022.md:5]: I would say "In the [Process Group](../mews-operations/reservations.md#process-group) request message, Channel is deprecated and replaced by Sources. Sources is a collection of new object [Source](../mews-operations/reservations.md#source), which provides additional information about reservation sources or channels."

Comment (MikeAdamsMews) [reply changelog/changelog2022.md:5]: Actually, Channel, Sources and Source should not be capitalized, so better to write them as `channel`, `sources` and `source`, I think that's the convention that's been used before to refer to fields or properties within messages.

File: changelog/changelog2022.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+* [Travel Agency](../mews-operations/reservations.md#travel-agency) was added to [Process Group Request](../mews-operations/reservations.md#request).
```
Comment (MikeAdamsMews) [inline changelog/changelog2022.md:6]: I would say "In the [Process Group](../mews-operations/reservations.md#process-group) request message, `company` is split into separate entities for `company` and `travelAgency`. `iata` is removed from `company` (deprecated). [`travelAgency`](../mews-operations/reservations.md#travel-agency) is a new object."

**Review by Vaclav-Hanselka — APPROVED** [review]

**Review by patrikmiksovsky95 — APPROVED** [review]

File: changelog/changelog2022.md [hunk 9]
```diff
@@ -0,0 +1,11 @@
+# Changelog 2022
+
+## 19th September 2022
```
Comment (MikeAdamsMews) [inline changelog/changelog2022.md:3]: ## 26th September 2022

* In [Process Group](../mews-operations/reservations.md#process-group), `channel` is deprecated and replaced by `sources`, which is a collection of new object [Source](../mews-operations/reservations.md#source) and provides additional information about reservation sources or channels.
* In [Process Group](../mews-operations/reservations.md#process-group), `company` is split into separate entities for [Company](../mews-operations/reservations.md#company) and [Travel Agency](../mews-operations/reservations.md#travel-agency). `iata` is removed from `company` (deprecated).

File: mews-operations/reservations.md [hunk 10]
```diff
@@ -248,6 +275,13 @@ The third `reservation` definition shows the partial cancellation - cancelling t
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `id` | `string` | optional | Identifier of company. |
+| `name` | `string` | optional | Name of company or travel agency. |
```
Comment (MikeAdamsMews) [inline mews-operations/reservations.md:278]: | ~~`iata`~~ | ~~`string`~~ | ~~optional~~ | ~~IATA number of travel agency.~~ Deprecated. |

File: mews-operations/reservations.md [hunk 11]
```diff
@@ -248,6 +275,13 @@ The third `reservation` definition shows the partial cancellation - cancelling t
...
+| `contact` | [`Contact`](#contact) object | optional | Company or travel agency contact information. |
+
```
Comment (MikeAdamsMews) [inline mews-operations/reservations.md:280]: | `name` | `string` | optional | Name of company. |
| `contact` | [`Contact`](#contact) object | optional | Company contact information. |

File: mews-operations/reservations.md [hunk 12]
```diff
@@ -248,6 +275,13 @@ The third `reservation` definition shows the partial cancellation - cancelling t
...
+#### Travel Agency
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
 | `iata` | `string` | optional | IATA number of travel agency. |
 | `name` | `string` | optional | Name of company or travel agency. |
```
Comment (MikeAdamsMews) [inline mews-operations/reservations.md:286]: | `name` | `string` | optional | Name of travel agency. |
| `contact` | [`Contact`](#contact) object | optional | Travel agency contact information. |

File: mews-operations/reservations.md [hunk 13]
```diff
@@ -288,12 +322,29 @@ The third `reservation` definition shows the partial cancellation - cancelling t
...
 | `13` | Maestro |
 
-#### Channel
+#### Source
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `code` | `int` | required | [Channel](#channels) code. |
-| `name` | `string` | required | Name of the Channel. |
+| `code` | `int` | required \(exc. new unknown sources\) | Source code. |
```
Comment (MikeAdamsMews) [inline mews-operations/reservations.md:329]: | `code` | `int` | required \(exc. new unknown sources\) | Code number of the source or channel. |

**Review by MikeAdamsMews — CHANGES_REQUESTED** [review]

**Review by MikeAdamsMews — APPROVED** [review]
Thank you for making those changes, it's looking good now.

**Pull request merged**